types/passport/index.d.ts

```diff
@@ -25,6 +26,9 @@ declare global {
             isAuthenticated(): boolean;
             isUnauthenticated(): boolean;
         }
+        interface User {
+            [_: string]: any;
```

Comment:
I still think you should consider adding documentation here to tell users how to use it optimally ;)

Reply:
He he, sorry I didn't do it, this is my first PR in definitelytyped and wasn't sure where the docs are, I'll have it in mind next time, thanks